### ISSUE TITLE
[Crash]: All the Mods 10  1.14 crash possibly related to Iron's Spellbooks

### ISSUE THREAD

**Issue by 1337phreak** [AllTheMods/ATM-10#991]
### Possible Fixes

Yes

### Modpack Version

1.14

### What happened?

I've had my dedicated server crash twice now and it seems to have happened users have used spells from the Iron's Spellbooks mod to kill mobs.

### Crash Log

https://gist.github.com/1337phreak/5fe4f3ee95254b2ff72dcc026e1a6392

### Latest.Log

https://gist.github.com/1337phreak/324ddc918f02024b4c1e5ae8f8d56e32

**Comment by AwakenedRose** [comment]
I'm also having this same issue, I'm noticing that the crash is happening when i cast a spell on an infernal mob . if need to ill share my crash log as well.

**Comment by 1337phreak** [comment]
I do believe for my server crashes the mob being attacked was augmented by the Infernal Mobs mod as well.

**Comment by Yogfan1** [comment]
I also had this happen though it was just a common Drowned... though it did have a backpack and some buffs. I had used poison arrow if that matters.

**Comment by Microjett** [comment]
I am running a ATM10 server Ver 1.14. We are also seeing server crashes whenever anyone uses a Iron's Spellbooks spell to damage any mob.

I've attached our latest log for people to verify, However this is happening in SP as well.

I loaded up a SP world in creative mode grabbed a irons spellbook spell scroll and fired it at some nearby cows and as soon as they took damage the game crashed.
[crash-2024-11-03_13.49.30-server.txt](https://github.com/user-attachments/files/17609086/crash-2024-11-03_13.49.30-server.txt)

This appears to be happening for any Irons spell that causes damage to another entity.
